cmake/modules/dts.cmake

```
@@ -53,6 +53,17 @@ list(APPEND
   ${SHIELD_DIRS}
   ${ZEPHYR_BASE}
   )
+
+# DTS directories can come from multiple places. Some places like a
+# user's CMakeLists.txt can preserve symbolic links. Others like
+# scripts/zephyr_module.py --settings-out resolve them.
+unset(_real_dts_root)
```

No reason for the leading `_`.

Generally we try to keep vars supposed to be used locally in lower case and vars to be globally used in upper case.

```suggestion
unset(real_dts_root)
```

---

Done, thx for the review!